en-us/install-instructions.md

```diff
@@ -89,6 +89,45 @@ For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Inst
 
 A simple installer for vanilla launcher [is coming soon](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/110)!
 
+### Pojav Launcher (Android)
+
+**This process is lengthy, may not work or may not improve performance for you. You've been warned.**
```

Comment:
```suggestion
**This process is lengthy, it may not work or may not improve performance for you. You've been warned.**
```
Is it worth saying that you may or may not receive support using this launcher? Since there's no PR for main repo, it won't be selectable in a bug report (ofc you could make one, but that's besides the point)